covid19dp_submission/nextflow/submission_workflow.nf

```diff
@@ -0,0 +1,125 @@
+process download_snapshot {
```

Comment:
I agree with Tim's earlier comment that this should be indented and formatted as per Nextflow documentation...  (I guess we need a Nextflow style guide as well as a Python one :sweat:)